service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1927,6 +1993,10 @@ components:
       items:
         $ref: "#/components/schemas/ConceptSetV2"
 
+    ConceptSetIdV2:
```

Comment:
We're going to need to audit at the "data feature set" level rather than the individual "concept set" ("data feature") level. Individual concept sets won't be top level objects anymore once we add "data feature sets". Does it make sense to flesh out that API first?

Reply:
RE top level objects: I'm not sure I'll implement "data feature sets" as actual collections of "concept sets" on the backend. I may just update the definition of a "concept set". Not sure though, since I haven't thought through all the changes, yet.

RE defining data feature sets API first: Since that's not an SDD prod requirement, I want to do the auditing stuff first. So my inclination here is to keep using concept sets for now.

Reply:
Per offline discussion -- we now think data features sets will be different enough from concept sets, that they'll probably warrant their own new schemas (i.e. the cohort definition schema isn't reusable enough for this case). So I removed the concept set activity types from this PR, given they're not required for SDD prod. I'll add in ones for data feature sets later post-prod.